app/jobs/build_submission_bundle_job.rb

```diff
@@ -9,12 +9,17 @@ def perform(submission_id)
 
     begin
       submission.generate_form_1040_pdf
-    rescue
-      submission.transition_to!(:failed, error_code: 'PDF-1040-FAIL')
+    rescue StandardError => e
+      submission.transition_to!(:failed, error_code: 'PDF-1040-FAIL', raw_response: ["Engineers should look in Sentry for an exception", "EfileSubmission #{submission_id}", e.class.name])
```

Comment:
The contents in raw_response outputs "raw"-ly in a <pre><code> block, so i think this may get output as an array. doesn't really matter, and i guess we can also loop through items in raw response if we want to format things more nicely, so just an FYI and not a blocker.

Reply:
@bytheway875 oh hah! I will adjust this to be plain text, then merge.

Reply:
Actually I'll re-request your review just to make sure we're on the same page. Thanks for a fruitful discussion.